internal/users/manager_test.go

```diff
@@ -168,10 +168,6 @@ func TestUpdateUser(t *testing.T) {
 		"Error_if_user_exists_on_system":                          {userCase: "user-exists-on-system", wantErr: true, noOutput: true},
 		"Error_if_group_exists_on_system":                         {groupsCase: "group-exists-on-system", wantErr: true, noOutput: true},
 
-		"Error_when_updating_local_groups_remove_user_from_db":                              {groupsCase: "mixed-groups-gpasswd-fail", localGroupsFile: "gpasswdfail_in_deleted_group.group", wantErr: true},
```

Comment:
As this is only a small diff, please ensure that you have a test when you remove last user from a local group that the groups remains (and that being explicitely written as a test case)

Reply:
I added a test case for that to `TestUpdateUser`.